### DISCUSSION TITLE
Is there a preset for standardjs / standard?

### DISCUSSION THREAD

**Discussion by theogravity** [biomejs/biome#2070] [Q&A] [answered]
My project currently uses StandardJS

https://standardjs.com/

Is there a preset that I can use that will conform to it?

**Comment by arendjr** [comment, marked as answer]
No, our formatter is based on the formatting from Prettier and our linter is more inspired by ESLint and its plugins. I’m afraid some of the choices from StandardJS are incompatible with it.
